app/controllers/poms_controller.rb

```diff
@@ -0,0 +1,14 @@
+class PomsController < ApplicationController
+  before_action :authorise
+
+  def show
+    staff_ids = params[:ids].split(',')
```

Comment:
Presumably :ids is a string of comma separated ids.  This is okay, but it's also possible (just out of interest) to pass the same parameter name multiple times, so that params[:id] is a list.  So for instance, you could do ..

id=1&id=2&id=3

Although this does reduce available characters when the query string length is limited.  I wouldn't change this yet though..